index.js

```diff
@@ -0,0 +1,59 @@
+const boardElement = document.getElementById('board')
+const resetButton = document.getElementById('reset')
+
+let board = ['', '', '', '', '', '', '', '', '']
+
+let x_turn = true;
+let winner = null;
+
+boardElement.addEventListener('click', function (event) {
+    if (event.target.classList.contains("block")) {
+        const blocks = Array.from(document.querySelectorAll(".block"))
+        const index = blocks.indexOf(event.target)
+
+        if (board[index] !== "" || winner !== null) return;
+
+        board[index] = x_turn ? "X" : "O";
+        event.target.textContent = board[index];
+
+        if (checkWin(board[index])) {
+            winner = board[index];
+            setTimeout(() => alert(`${winner} wins!`), 100);
+            return;
+        }
+
+        // 🚨 Check for a draw
+        if (!board.includes("")) {
+            setTimeout(() => alert("It's a draw!"), 100);
+            return;
+        }
+
+        x_turn = !x_turn
+    }
+})
+
+resetButton.addEventListener('click', function () {
```

Comment:
Rename the variables and functions appropriately. The button we have right now is `Restart` so it should be `restartButton`.